RickAndMorty/SceneDelegate.swift

```diff
@@ -11,16 +11,16 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
 
     var window: UIWindow?
 
-    let coordinator = DefaultTabControllerCoordinator()
+    //let coordinator = DefaultTabControllerCoordinator()
```

Comment:
Is this comment necessary for the code ?
If so, let a TODO comment or just remove it